documentation/release_notes.rst

```diff
@@ -8,6 +8,33 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6.1
+===============
+
+Fixed Issues
+------------
+- Fixed compilation errors happening when using C++20.
+- Fixed ``CL_OUT_OF_RESOURCES`` issue which used to occur during execution of Radix sort algorithm on CPU devices.
+- Fixed an issue with the following algorithms used with DPC++ execution policies and the data located in the device memory (USM): ``exclusive_scan_by_segment``, ``inclusive_scan_by_segment``, ``reduce_by_segment``.
```

Comment:
1) the line is too long, needs to be split
2) what sort of an issue, how would it be observed?

Reply:
Given the initial wording of "Fixed error in ``exclusive_scan_by_segment`` algorithm when used with unified shared device memory (USM) and number of items equals to 1", claiming this to be "added support" sounds too ambitious :)

How about "Fixed crashes in ``exclusive_scan_by_segment``, ``inclusive_scan_by_segment``, ``reduce_by_segment`` algorithms applied to device-allocated USM"?